train/train.py

```diff
@@ -48,27 +50,75 @@ def self_play(model_path, out_dir, config):
                    stderr=sys.stderr, stdout=sys.stdout, check=True)
 
 
-def train(model_path, data_dir, config):
-    pass  # TODO
+def train(model_path, _, config):
+    # TODO change
+    data_dir = Path(config["data_dir"])
+
+    def reverse_position(pos, winner):
+        return [-x for x in pos], -winner
+
+    positions = []
+    winners = []
+
+    logging.debug('a')
+
+    # Load all games made by the current model
+    for training_iteration_dir in data_dir.iterdir():
+        if str(model_id(model_path)) not in str(training_iteration_dir):
+            continue
+        for data_file in training_iteration_dir.iterdir():
+            with open(os.path.join(data_dir, data_file), "rb") as f:
+                data_obj = json.load(f)
+            pos, win = data_obj["position"], data_obj["winner"]
+            if random.choice([True, False]):

```

Comment:
we should remove it, and always display the position to the network as it's turn and turn=1, see hex_utils.load_data_entry